docs/source/trainer/artifact_logging.rst

```diff
@@ -0,0 +1,96 @@
+|:hindu_temple:| Artifact Logging
+====================
+
+What is an artifact?
+-----------
+
+An artifact can be thought of as a folder stored in the cloud. This folder can contain any
+files or file URL that are pertinent to a training run. These files are called **assets**
+or **asset files**. Assets can be any type of inputs to or byproducts of a training run,
+like datasets, checkpoints, config files, etc. This folder different from a normal folder
+or directory you would have on your local system is that it is ‘versioned’ and it is often
```

Comment:
Artifacts are not versioned unless if the backend supports versioning. WandB versions automatically, and s3 versions if versioning is enabled by default in the bucket. Composer does not rely on versioning.